text/0635-mediaconvert-l2-construct.md

```diff
@@ -0,0 +1,258 @@
+
+
+### Customer Usage and Interaction
+
+#### Job Templates
+
+To create a Custom Job Template using an L2 Construct, you define the ``jobTemplateProps`` object, which includes all the 
+necessary settings for your media processing tasks. This object is then passed to the ``JobTemplate`` construct to create the custom template.
+
+    const jobTemplateProps: JobTemplateProps = {
+      name: 'CustomJobTemplate',
+      settings: {
+        OutputGroups: [
+          {
+            Name: 'CustomOutputGroup',
+            OutputGroupSettings: {
+              Type: 'FILE_GROUP_SETTINGS',
+              FileGroupSettings: {
+                Destination: 's3://amzn-s3-demo-destination-bucket/'
+              }
+            },
+            Outputs: [
+              {
+                ContainerSettings: {
+                  Container: 'MP4'
+                },
+                VideoDescription: {
+                  CodecSettings: {
+                    Codec: 'H_264',
+                    H264Settings: {
+                      MaxBitrate: 1000,
+                      RateControlMode: 'QVBR',
+                      SceneChangeDetect: 'TRANSITION_DETECTION'
+                    }
+                  }
+                },
+                AudioDescriptions: [
+                  {
+                    CodecSettings: {
+                      Codec: 'AAC',
+                      AacSettings: {
+                        Bitrate: 96000,
+                        CodingMode: 'CODING_MODE_2_0',
+                        SampleRate: 48000
+                      }
+                    }
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    };
+
+    // Define the custom MediaConvert Job Template
+    const customJobTemplate = new JobTemplate(this, 'CustomJobTemplate', jobTemplateProps);
+
+In this example, the ``customJobTemplate`` is configured with specific output groups and encoding settings. By defining 
+these properties, you can create a Job Template that aligns precisely with your media processing requirements, ensuring 
+consistent and optimized workflows across your jobs.
+
+The System Job Templates are a collection of pre-configured and managed JobTemplates provided by AWS Elemental MediaConvert. 
+These templates are designed to cater to common media processing use cases and are set up with best practices in mind. 
+By utilizing these System Job Templates, you can easily configure your MediaConvert jobs by simply referencing the template's 
+name. The necessary settings and configurations will be automatically applied, ensuring that your media processing workflows 
+are optimized and aligned with recommended standards.
+
+    const genericMp4Hev1AvcAacSdrQvbJobTemplate = new JobTemplate(this, 'GenericMp4Hev1AvcAacSdrQvbJobTemplate', {
+        name: "GenericMp4Hev1AvcAacSdrQvbJobTemplate"
```

Comment:
where are these templated maintained?

Reply:
My initial idea is to maintain (a copy of these system templates) inside CDK repo. Discussed offline with @moelasmar and pending implementation details.